sdk/identity/identity/src/credentials/vscodeCredential.ts

```diff
@@ -19,21 +19,34 @@ const AzureAccountClientId = "aebc6443-996d-45c2-90f0-388ff96faa56"; // VSC: 'ae
 const VSCodeUserName = "VS Code Azure";
 const logger = credentialLogger("VSCodeCredential");
 
+/**
+ * Provides options to configure the Visual Studio Code credential.
+ */
+export interface VSCodeCredentialOptions extends TokenCredentialOptions {
+    tenantId?: string;
+}
+
 /**
  * Connect to Azure using the credential provided by the VSCode extension 'Azure Account'.
  * Once the user has logged in via the extension, this credential can share the same refresh token
  * that is cached by the extension.
  */
 export class VSCodeCredential implements TokenCredential {
   private identityClient: IdentityClient;
+  private tenantId: string;
 
   /**
    * Creates an instance of VSCodeCredential to use for automatically authenicating via VSCode.
    *
    * @param options Options for configuring the client which makes the authentication request.
    */
-  constructor(options?: TokenCredentialOptions) {
+  constructor(options?: VSCodeCredentialOptions) {
     this.identityClient = new IdentityClient(options);
+    if (options && options.tenantId) {
+      this.tenantId = options.tenantId;
+    } else {
+      this.tenantId = CommonTenantId;
```

Comment:
I don't know that we should be setting this to "common" in the case it's not specified. If I recall correctly I think VS Code stores the tenantId in it's configuration, which is what we should use as the default if it's not specified.

Reply:
I can file an issue so we can revisit this post-release. Seems like the right plan would be to use more of the user's configuration.